waku/v2/node/peer_manager/waku_peer_store.nim

```diff
@@ -57,3 +59,18 @@ proc peers*(peerStore: WakuPeerStore): seq[StoredInfo] =
                        toSeq(keys(peerStore.keyBook.book))).toHashSet()
 
   return allKeys.mapIt(peerStore.get(it))
+
+
+proc deletePeer*(peerStore: WakuPeerStore, peerId: PeerID) =
```

Comment:
What happens to a deleted peer after a restart? Wouldn't it just be loaded again from persistent storage (where that is configured)? This may be the behaviour we want for now - i.e. we don't "delete" blacklisted peers, just keep track of their blacklisted status?

Reply:
We just want to keep track of the blacklisted peer, so persisting the blacklist should solve this.

Reply:
On this note, we aren't really deleting a peer, more like removing it from peers we try to connect to (e.g.)